test/CadlRanchProjects.Tests/MgmtTestHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Azure.Core;
+using Azure.ResourceManager;
+using static CadlRanchProjects.Tests.OAuth2TestHelper;
+
+namespace CadlRanchProjects.Tests
+{
+    public class MgmtTestHelper
+    {
+        public static ArmClient CreateArmClientWithMockAuth(Uri host, string defaultSubscriptionId = default)
+        {
+            var options = new ArmClientOptions();
+            options.AddPolicy(new MockBearerTokenAuthenticationPolicy(new MockCredential(), new string[] { "https://security.microsoft.com/.default" }, options.Transport), HttpPipelinePosition.PerCall);
```

Comment:
since this is mock, do we still need a valid scope and audience?

Reply:
1. We are not testing auth here, so no one will read these things.
2. Even we are testing auth, what we need to test is not "what is a valid scope", but "if the scope user sets is sent to server without change", and we already have that [test](https://github.com/Azure/autorest.csharp/blob/72659d42a5067035f0c2d969ed07dd523cbda4dc/test/CadlRanchProjects.Tests/authentication-oauth2.cs#L18).

Reply:
I meant we can use test string for these things.